congame-doc/introduction.scrbl

```diff
@@ -87,40 +92,77 @@ an @racket[md] expression to denote text that will be formatted using Markdown.}
 
 @;~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-@; @subsection{Previewing the study}
+@subsection{Test-driving the simplest study}
+
+To try out your study, you’ll need to upload it to a Congame server and enroll in an @tech{instance}
+of the study. The Docker container you set up during the initial
+@seclink["install-congame"]{installation process} provides the Congame server for use in testing.
+
+Open the Docker desktop app and press the play ▶️  button for the @tt{congame} container.
+
+In DrRacket, save the study you created above as @filepath{simple.rkt}.
+
+Then click the @onscreen{Upload Study} button:
+
+@screenshot{intro-drr-upload-btn.png}
+
+@subsubsection{Aside: one-time connection setup}
+
+The very first time you upload a Conscript study from DrRacket, DrRacket will ask you the address of
+the Congame server to connect to, and then prompt you to log in on that server.
+
+DrRacket assumes the server you want is at @tt{http://localhost:5100} --- that is, is the one
+running on the Docker container on your computer. No need to change it, so click the @onscreen{Log
+in} button. Log in using the built-in user @tt{admin@"@"congame.local} with password @tt{admin}.
+
+@margin-note{@mark{If you later want DrRacket to send uploads to a different server, too bad.}}
```

Comment:
This should be fixed in current version, but test.

Reply:
So you start off with docker running, then you upload and get some connection, then you change the server to local dev after which you get the "connection failed"? Adding @Bogdanp .

Reply:
Currently, we only ask the user to log in again when the server returns a 401 response. If the server is down, we just show an error. We can take a look at that next session.

Reply:
It's not a big deal, I just raise it as something that might become an issue. To get my screenshots, I'll see if I can manually clear the preference somehow.